source/patterns/@aws-solutions-constructs/aws-lambda-eventbridge/lib/index.ts

```diff
@@ -0,0 +1,135 @@
+   */
+  readonly existingVpc?: ec2.IVpc;
+  /**
+   * Properties to override default properties if deployVpc is true
+   */
+  readonly vpcProps?: ec2.VpcProps;
+  /**
+   * Whether to deploy a new VPC
+   *
+   * @default - false
+   */
+  readonly deployVpc?: boolean;
+  /**
+   * Optional Name for the EventBus's name environment variable set for the Lambda function.
+   *
+   * @default - EVENTBUS_NAME is used
+   */
+  readonly eventBusEnvironmentVariableName?: string;
+}
+
+/**
+ * @summary The LambdaToEventbridge class.
+ */
+export class LambdaToEventbridge extends Construct {
+  public readonly lambdaFunction: lambda.Function;
+  public readonly eventBus?: events.IEventBus;
+  public readonly vpc?: ec2.IVpc;
+
+  /**
+   * @summary Constructs a new instance of the LambdaToEventbridge class.
+   * @param {cdk.App} scope - represents the scope for all the resources.
+   * @param {string} id - this is a a scope-unique id.
+   * @param {LambdaToEventbridgeProps} props - user provided props for the construct.
+   * @since 1.120.0
+   * @access public
+   */
+  constructor(scope: Construct, id: string, props: LambdaToEventbridgeProps) {
+    super(scope, id);
+    defaults.CheckProps(props);
+
+    if (props.deployVpc || props.existingVpc) {
+      if (props.deployVpc && props.existingVpc) {
+        throw new Error("More than 1 VPC specified in the properties");
+      }
+
+      this.vpc = defaults.buildVpc(scope, {
+        defaultVpcProps: defaults.DefaultIsolatedVpcProps(),
+        existingVpc: props.existingVpc,
+        userVpcProps: props.vpcProps,
+        constructVpcProps: {
+          enableDnsHostnames: true,
+          enableDnsSupport: true,
+        },
+      });
+      defaults.AddAwsServiceEndpoint(scope, this.vpc, defaults.ServiceEndpointTypes.EVENTS);
+    }
+
+    // Setup the Lambda function
+    this.lambdaFunction = defaults.buildLambdaFunction(this, {
+      existingLambdaObj: props.existingLambdaObj,
+      lambdaFunctionProps: props.lambdaFunctionProps,
+      vpc: this.vpc,
+    });
+
+    // Build Event Bus
+    this.eventBus = defaults.buildEventBus(this, {
+      existingEventBusInterface: props.existingEventBusInterface,
+      eventBusProps: props.eventBusProps
+    });
+
+    // Configure environment variables
+    const eventBusEnvironmentVariableName = props.eventBusEnvironmentVariableName || 'EVENTBUS_NAME';
+    const eventBusName = this.eventBus?.eventBusName || 'default';
+    this.lambdaFunction.addEnvironment(eventBusEnvironmentVariableName, eventBusName);
+
+    // Enable putevents permissions for the Lambda function by default
+    if (this.eventBus) {
+      this.eventBus.grantPutEventsTo(this.lambdaFunction);
+    } else {
+      // Since user didn't specify custom event bus, provide permissions on default event bus
+      const defaultEventBus = events.EventBus.fromEventBusName(this, 'default-event-bus', 'default');
```

Comment:
Wait - is "default" the actual name? That makes it much easier.

Reply:
each region has a `default` eventBus.